assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/useBodyRowGenerator.tsx

```diff
@@ -85,12 +88,17 @@ const useBodyRowGenerator = (): DatesTableBodyRowGen => {
 			{
 				key: 'actions',
 				type: 'cell',
-				className: 'ee-date-list-cell ee-actions-column ee-rspnsv-table-column-big',
+				className: 'ee-date-list-cell ee-date-list-col-actions ee-actions-column ee-rspnsv-table-column-big',
 				value: sortingEnabled ? '-' : <DateActionsMenu entity={datetime} />,
 			},
 		];
 
-		const cells = cellsData.filter(filterCellByStartOrEndDate(displayStartOrEndDate));
+		const exclude = ['row', 'stripe', 'name', 'actions'];
+
+		const cells = pipe(
+			filter(filterCellByStartOrEndDate(displayStartOrEndDate)),
+			addZebraStripesOnMobile(exclude)
+		)(cellsData);
```

Comment:
added required changes

Reply:
that `addZebraStripesOnMobile(exclude)` is applied only once after filtering is done, it's not every item filtering

Reply:
your observation is good about function creation, because since it's executed out of the component, it will be initialized only once; a minor perf gain, but still important to keep in mind

Reply:
yes it is minor... but here's something to consider:

- we run EE on EventSmart

- EventSmart has... i think 50K + sites on it that are all running on the same server cluster (don't quote me on that number though, I may be wildly off)

- we pay for all of the resources that the above server cluster uses

- a small performance gain on a page  X  number of requests for that page per day  X  365 days a year  X  cost per CPU hour  =  $$$ savings per year


that savings amount may not be that significant for one single optimization, but a culture of always trying to write performant code can all add up over time and make a big difference than if optimization was only considered for "BIG" things

Reply:
agree, need to incorporate also measuring and profiling more as a habit or smth integrated in workflow

Reply:
this led me to this: https://github.com/GoogleChromeLabs/lighthousebot